azure-cli2017.pyproj

```diff
@@ -1769,16 +1784,7 @@
     <Content Include="command_modules\azure-cli-vm\linter_exclusions.yml" />
   </ItemGroup>
   <ItemGroup>
-    <Interpreter Include="..\..\env2\">
-      <Id>env2</Id>
-      <Version>2.7</Version>
-      <Description>env2 (Python 2.7 (64-bit))</Description>
-      <InterpreterPath>Scripts\python.exe</InterpreterPath>
-      <WindowsInterpreterPath>Scripts\pythonw.exe</WindowsInterpreterPath>
-      <PathEnvironmentVariable>PYTHONPATH</PathEnvironmentVariable>
-      <Architecture>X64</Architecture>
-    </Interpreter>
-    <Interpreter Include="..\..\env\">
+    <Interpreter Include="..\env\">
```

Comment:
You don't need to change that. Each developer may have different env setting.

Reply:
fixed.